### PR TITLE
Add "Klassen" exercise

### PR DESCRIPTION
**Pull request by akalcik**

### CODE REVIEW

**Comment by akalcik** [comment]
[Übung C#: Klassen und Methoden](https://trello.com/c/BPtNbq7c/66-%C3%BCbung-c-klassen-und-methoden)

File: csharp/Klassen/Readme.md [hunk 1]
```diff
@@ -44,3 +49,146 @@ public class Freund
...
+        }
+    }
+
+    static void Main(string[] args)
+    {
+        Freund neueFreund = new Freund("Peter");
+        neueFreund.Alter = 8;
+
+        neueFreund.SagHallo();
+        Console.WriteLine("Hallo " + neueFreund.Name);
```
Comment (rigerta) [inline csharp/Klassen/Readme.md]: Console.ReadKey(); is missing.

Comment (akalcik) [reply csharp/Klassen/Readme.md]: Yeah this is generally a problem. When you run it out of the Visual Studio on debug mode, it'll stop on his own. This is only the issue when your run it not in debug mode I think. We leave as is, wen can than discuss it with the children.

**Comment by kwaxi** [comment]
In C# ist der default access modifier ```internal``` - siehe https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers

**Pull request merged**